x-pack/plugins/security_solution/public/flyout/right/components/analyzer_preview.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useEffect, useState } from 'react';
+import type { TimelineEventsDetailsItem } from '../../../../common/search_strategy/timeline';
+import { useAlertPrevalenceFromProcessTree } from '../../../common/containers/alerts/use_alert_prevalence_from_process_tree';
+import type { StatsNode } from '../../../common/containers/alerts/use_alert_prevalence_from_process_tree';
+import { AnalyzerTree } from './analyzer_tree';
+import { isActiveTimeline } from '../../../helpers';
+
+interface Cache {
+  statsNodes: StatsNode[];
+}
+interface AnalyzerPreviewProps {
+  /**
+   * Entity id details from ecs data
+   */
+  entityId: TimelineEventsDetailsItem;
+  /**
+   * Document id details from ecs data
+   */
+  documentId: TimelineEventsDetailsItem;
+  /**
+   * Index details from ecs data
+   */
+  index: TimelineEventsDetailsItem;
+  /**
+   * Data test subject string
+   */
+  ['data-test-subj']: string;
+}
+
+/**
+ * Analyzer preview under Overview, Visualizations. It shows a tree representation of analyzer.
+ */
+export const AnalyzerPreview: React.FC<AnalyzerPreviewProps> = ({
+  entityId,
+  documentId,
+  index,
+  'data-test-subj': dataTestSuj,
+}) => {
+  const scopeId = 'fly-out'; // TO-DO: change to scopeId from context
+  const [cache, setCache] = useState<Partial<Cache>>({});
+
+  const { values: wrappedProcessEntityId } = entityId;
+  const processEntityId = Array.isArray(wrappedProcessEntityId) ? wrappedProcessEntityId[0] : '';
+  const { values: wrappedDocumentId } = documentId;
+  const processDocumentId = Array.isArray(wrappedDocumentId) ? wrappedDocumentId[0] : '';
+  const { values: indices } = index;
+
+  const { loading, error, statsNodes } = useAlertPrevalenceFromProcessTree({
+    processEntityId,
```

Comment:
looking at the code in the [useAlertPrevalenceFromProcessTree](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/common/containers/alerts/use_alert_prevalence_from_process_tree.ts#L95), the `processEntityId` isn't even used, so we could do the following:
- refactor the code there and remove it (only if it isn't a big effort)
- just pass an empty string from here and remove the code related to `entityId` above and in the parent

Thoughts?